### DISCUSSION TITLE
Unknown network error attempting to reach coklj8ay.api.sanity.io

### DISCUSSION THREAD

**Discussion by Pranav-Bobde** [sanity-io/sanity#4056] [Sanity Studio Support]
I created a new project using the command on the site:
```bash
npm create sanity@latest -- --template clean --create-project "Sanity Project" --dataset production
```

and then inside the project directory ran: npm run dev

And this is the error that I'm getting:

![image](https://user-images.githubusercontent.com/66965591/211331209-adf87434-d8f6-4f55-87d6-f04c36e42d3e.png)

**Comment by cthulhuscode** [comment]
Check for extensions that might be blocking access to the page, like Privacy Badger, uBlock Origin, HTTPS Everywhere... Disabled them, I had the same error and solved it by disabling them.
